File: Condicionales.py
Figura= input('selecione la figura a calcular su area: \n\n 1 para rombo\n\n 2 para rectangulo \n\n 3 para cuadrado \n\n 4 para trapecio \n\n: ')
Pi=3.1416
const= int(2)
match Figura:
    case '1':
        Dmayor= int(input('ingresa el valor de la diagonal mayor: '))
        Dmenor= int(input('ingresa el valor de la diagonal menor: '))
        Area=(Dmayor+Dmenor/const)
        print('El area del rombo es: ',Area)
    case '2':
        Largo= int(input('ingresa el valor del largo del rectangulo: '))
        Ancho= int(input('ingresa el valor del ancho del rectangulo: '))
        Area=Largo*Ancho
        print('El area del rectangulo es: ',Area)
    case '3':
        Lado= int(input('ingresa el valor del lado del cuadrado: '))
        Area=Lado*Lado
        print('El area del cuadrado es: ',Area)         
    case '4':
        Bmayor= int(input('ingresa el valor de la base mayor: '))
        Bmenor= int(input('ingresa el valor de la base menor: '))
        H=int (input('ingrese la altura del trapecio: '))
        Area=(((Bmayor+Bmenor)*H)/2)
        print('El area del trapecio es: ',Area)

File: Anyelo Carrillo Java/condicionales.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Seleccione la figura a calcular su área:\n\n 1 para rombo\n\n 2 para rectángulo\n\n 3 para cuadrado\n\n 4 para trapecio\n\n: ");
        String Figura = scanner.nextLine();

        double Pi = 3.1416;
        int const = 2;

        switch (Figura) {
            case "1":
                System.out.print("Ingresa el valor de la diagonal mayor: ");
                int Dmayor = scanner.nextInt();
                System.out.print("Ingresa el valor de la diagonal menor: ");
                int Dmenor = scanner.nextInt();
                double AreaRombo = (Dmayor + Dmenor) / const;
                System.out.println("El área del rombo es: " + AreaRombo);
                break;
            case "2":
                System.out.print("Ingresa el valor del largo del rectángulo: ");
                int Largo = scanner.nextInt();
                System.out.print("Ingresa el valor del ancho del rectángulo: ");
                int Ancho = scanner.nextInt();
                int AreaRectangulo = Largo * Ancho;
                System.out.println("El área del rectángulo es: " + AreaRectangulo);
                break;
            case "3":
                System.out.print("Ingresa el valor del lado del cuadrado: ");
                int Lado = scanner.nextInt();
                int AreaCuadrado = Lado * Lado;
                System.out.println("El área del cuadrado es: " + AreaCuadrado);
                break;
            case "4":
                System.out.print("Ingresa el valor de la base mayor: ");
                int Bmayor = scanner.nextInt();
                System.out.print("Ingresa el valor de la base menor: ");
                int Bmenor = scanner.nextInt();
                System.out.print("Ingrese la altura del trapecio: ");
                int H = scanner.nextInt();
                double AreaTrapecio = ((Bmayor + Bmenor) * H) / 2.0;
                System.out.println("El área del trapecio es: " + AreaTrapecio);
                break;
            default:
                System.out.println("Opción no válida.");
        }
    }
}
